is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
+  array.forEach((currentElement, currentIndex) => {
+    array.slice(currentIndex + 1).forEach((elementToCompare) => {
```

Comment:
Can we avoid having a nested loop if possible? Maybe we can first populate a hash with the elements and then use that to lookup each element.

Reply:
Is it possible to return early if we find a duplicate element?